paddle/fluid/pybind/eager_functions.cc

```diff
@@ -333,8 +333,7 @@ static std::vector<paddle::any> CastAttrsToTragetType(
                         src.size()));
   for (size_t i = 0; i < src.size(); i++) {
     size_t end = attrs_names[i].find(": ");
-    std::string type_name =
-        attrs_names[i].substr(end + 2, attrs_names.size() - end - 2);
```

Comment:
`attrs_names.size()` should be `attrs_names[i].size()`

Reply:
My bad